app/queries/draft_and_live_versions.rb

```diff
@@ -0,0 +1,56 @@
+module Queries
+  module DraftAndLiveVersions
+    extend ArelHelpers
+    def self.call(content_item, target_class, locale = nil, base_path = nil)
+      content_items_table = table(:content_items)
+      states_table = table(:states)
+      translations_table = table(:translations)
+      locations_table = table(:locations)
+      versions_table = table(target_class)
+      versions_fk = target_class == "lock_versions" ? :target_id : :content_item_id
+
+      if base_path.nil?
+        base_path = locations_table.project(locations_table[:base_path])
+          .where(locations_table[:content_item_id].eq(content_item.id))
+      end
```

Comment:
I believe that was actually wrong
e.g. 
```ruby
 content_id = SecureRandom.uuid
FactoryGirl.create(:live_content_item, content_id: content_id, user_facing_version: 3, base_path: "/old-base-path")
draft = FactoryGirl.create(:draft_content_item, content_id: content_id, user_facing_version: 2, base_path: "/new-base-path")
Queries::DraftAndLiveVersions.call(draft, :user_facing_versions, "en")
```
`=> {"draft"=>2}`

Though I don't think it'll have come into view at all as you'd need to have a incorrect version at the same time as a new content item changes it's base_path